exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((res) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    });
+
+    //Asynchronous Error Handling
+    Promise.resolve().then(() => {
+        throw new Error('Broken')
+    }).catch(next)
+});
+
+//DELETE Route (/deletePerson/:id)
+app.delete('/deletePerson/:id', (req, res) => {
+    const { id } = req.params;
+
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id=1', id)
```

Comment:
The 'id' should not be hardcoded to equal 1 in the URL field. It should be able to change according the variable 'id' that was made on line 25.

Reply:
Agree with Jonathan here as well

Reply:
> The 'id' should not be hardcoded to equal 1 in the URL field. It should be able to change according the variable 'id' that was made on line 25.

I fixed the issue, not completely sure if its the right way.